### DISCUSSION TITLE
Boarding pod does not fit in ammo shelves

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#14891] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

the boarding pod item can't be placed in item shelves
it's missing whatever tags necessary to do this
it should be fittable, as it is alongside depth charges

### Reproduction steps

1. spawn boarding pod
2. try to store it in an ammo shelf

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This is intentional: the item is clearly so much larger than depth charges that it wouldn't fit in the shelves.

**Comment by Ek-Videogames** [comment]
to be honest... I kind of just thought it would use the same 'contained' sprite

- however on a technical level I can see how it'd be annoying to code it to not be enterable when it's just on random shelves.

**Reply by Regalis11** [reply]
I've personally imagined that the loader is embedded in a wall, and that there's space behind the opening of the loader where the pod can fit. But on a shelf, there's clearly no way for it to fit, so I would find it odd if you cn put it on one.ould
